pyproject.toml

```diff
@@ -70,3 +72,7 @@ exclude = [
   "mlflow/temporary_db_migrations_for_pre_1_users",
   "tests/protos",
 ]
+[tool.ruff.per-file-ignores]
+"dev/*" = ["T201", "T203"]
+"tests/resources/example_project/greeter.py" = ["T201", "T203"]
+"tests/resources/mlflow-test-plugin/mlflow_test_plugin/fake_deployment_plugin.py" = ["T201", "T203"]
```

Comment:
https://github.com/mlflow/mlflow/pull/9010/files#r1257466602 same comment on this file